comment/tests/test_utils.py

```diff
@@ -337,36 +336,34 @@ def test_for_drf(self):
         self.assertEqual(EmailInfo.CONFIRMATION_SENT, response)
 
 
-class UtilsTest(TestCase):
+class TestUtils:
     """Test general purpose utilities that aren't necessarily related to a comment"""
-
-    def setUp(self):
-        self.len_id = 6
+    len_id = 6
 
     def test_id_generator_length(self):
-        self.assertEqual(self.len_id, len(id_generator()))
+        assert self.len_id == len(id_generator())
 
     def test_id_generator_generates_different_ids(self):
-        self.assertNotEqual(id_generator(), id_generator())
+        assert id_generator() != id_generator()
```

Comment:
can you please change this test as suggested to get rid of SonarLint error
```suggestion
        first_id = id_generator()
        second_id = id_generator()
        assert first_id != second_id
```